packages/subgraph/tests/token/token-voting.test.ts

```diff
@@ -251,3 +266,87 @@ test('Run TokenVoting (handleVotingSettingsUpdated) mappings with mock event', (
 
   clearStore();
 });
+describe('handleMembershipContractAnnounced', () => {
+  beforeAll(() => {
+    // mock get balance
+    getBalanceOf(CONTRACT_ADDRESS, CONTRACT_ADDRESS, ERC20_AMOUNT_HALF);
+    getBalanceOf(
+      WRAPPED_CONTRACT_ADDRESS,
+      WRAPPED_CONTRACT_ADDRESS,
+      ERC20_AMOUNT_HALF
+    );
+    // create token calls
+    createTokenCalls(
+      CONTRACT_ADDRESS,
+      'Test Token',
+      'TT',
+      '18',
+      ERC20_AMOUNT_FULL
+    );
+    createWrappedTokenCalls(
```

Comment:
this should be a mockCall_createWrappedTokenCalls
similar to https://github.com/aragon/osx/blob/develop/packages/subgraph/tests/helpers/method-classes.ts#L160